File: P1/E1/BicicletaMontana.java
package E1;

public class BicicletaMontana extends Bicicleta {

    BicicletaMontana(int id){
        super(id);
    }

    @Override
    public String correr(){
        return "Corre una bicicleta de montaña de id: " + super.id;
    }
}

File: P1/E2/BicicletaMontana.py
from E2.Bicicleta import Bicicleta

class BicicletaMontana(Bicicleta):
    def __init__(self, id):
        super().__init__(id)

    def pedalear(self):
        print("Corre una bicicleta de montaña de id:", self.id)
